### ISSUE TITLE
Add the valabuilder to scons

### ISSUE THREAD

**Issue by bdbaddog** [SCons/scons#2147]
This issue was originally created at: 2008-07-20 04:10:38.
This issue was reported by: `piccaf`.
piccaf said at 2008-07-20 04:10:39
>Hello
> 
>Is it possible to add the valaBuilder that you can find in the wiki, so scons
>can work out of the box with vala files.
> 
>thanks,
> 
>frederic

gregnoel said at 2008-07-29 06:00:40
>Bug party triage.  Depends on implementing the batch builder mode.

gregnoel said at 2008-12-26 13:30:38
>Adjust triage of issues.

stevenknight said at 2009-01-09 09:43:49
>Batch build action support (issue 1086) has been added, so it should now be 
>possible to address this properly.
> 
>For easier navigation, the builder page on the wiki is:
> 
>http://www.scons.org/wiki/ValaBuilder
> 

dsvensson said at 2009-01-11 15:40:12
>A newer version can be found here:
> 
>http://abraca.xmms.se/browser/site_scons/site_tools/vala.py

stevenknight said at 2009-11-10 18:00:19
>stevenknight => issues@scons

russel said at 2010-05-06 11:18:01
>http://abraca.xmms.se/browser/site_scons/site_tools/vala.py
> 
>Now gives a 404?
> 
>The builder at http://www.scons.org/wiki/ValaBuilder assumes valac is a prepass 
>creating a .c and .h file.  As far as I can see valac will actually manage a complete 
>compile and link such that the .c and .h file are never made visible in the user 
>filestore.
> 
>Issue 2621 seems effectively to be duplicate of this one so which one to go with?
> 
>I propose starting a tool development using Bazaar and Launchpad as per 
>http://www.scons.org/wiki/ToolsIndex

russel said at 2010-05-06 23:13:58
>Bazaar branch of a Vala tool duly started and stored at Launchpad 
>https://code.launchpad.net/~russel/sconsaddons/vala

gregnoel said at 2010-05-19 08:07:22
>Bug party triage.  Development of new tools is done by a community effort.  See
>http://scons.org/wiki/ToolsIndex for details; contact Russel Winder directly to
>get involved in the effort. 

russel said at 2012-01-03 05:24:03
>Just for information the Launchpad Bazaar repository got transformed to a 
>Mercurial repository on BitBucket:  https://bitbucket.org/russel/scons_vala

gregnoel said this issue depends on #1086 at 2008-07-29 06:00:40.
